docs/calico-with-docker/without-docker-networking/README.md

```diff
@@ -63,13 +63,25 @@ You should see output like this on each node
 
 If you are not running in the cloud you may skip this step and jump to step 4.
 
-If you are running in the cloud, you will need to first configure an IP Pool
+If you are running in a cloud other than AWS, you will need to first configure an IP Pool
```

Comment:
I think it would look nicer if you had two subsections now that the cloud services would use different commands.

Maybe change the main section to just be **Running in the Cloud**, then have subsections of **GCE / Digital Ocean**  and **AWS**